CHANGELOG.unreleased.md

```diff
@@ -27,3 +29,5 @@
 > - major: if the change breaks compatibility
 >
 > In case of conflict, the highest (lowest in previous list) `$version` wins.
+
+- xo-web minor
```

Comment:
patch?

Reply:
`patch: if the change is a bug fix or a simple code improvement`

Reply:
IMHO, it's not a code improvement(refactoring) but a new functionality to `XO`.